src/script/standard.h

```diff
@@ -70,15 +81,49 @@ class CNoDestination {
  */
 typedef boost::variant<CNoDestination, CKeyID, CScriptID> CTxDestination;
 
+/** Get the name of a txnouttype as a C string, or nullptr if unknown. */
 const char* GetTxnOutputType(txnouttype t);
 
+/**
+ * Parse a scriptPubKey and identify script type for standard scripts. Assigns
+ * the result to the typeRet reference parameter and returns true if successful.
```

Comment:
Looks like you lost the stuff noting what vSolutionsRet is when you moved the comment from standard.cpp.